docs/pages/desktop-access/getting-started.mdx

```diff
@@ -236,7 +236,7 @@ dsacls "CN=NTAuthCertificates,CN=Public Key Services,CN=Services,CN=Configuratio
 
 ## Step 4/6. Export your Domain Controller's CA Certificate
 
-Teleport connects to your Domain Controller via LDAPS. This means that you must let Teleport know that the certificate sent 
+Teleport connects to your Domain Controller via LDAPS. This means that you must let Teleport know that the certificate sent
 by your Domain Controller during the initial SSL connection is trusted. If your Domain Controller's certificate is trusted by
 the system repository on the system running Teleport, you can skip this step. Atlernately, if you trust your network, you can
```

Comment:
We should make a few small tweaks here:

1. We never actually mention the config field that contains the LDAPS CA certificate.
2. Typo in `Alternately` - we should say `Alternatively`
3. I wouldn't say "if you trust your network" for the `insecure_skip_verify` part. We don't want to encourage the use of this setting for anything other than initial setup or POCs. Maybe say something like "If you are unable to acquire the LDAP CA certificate, you can skip TLS verification by setting `insecure_skip_verify: true`. We do not recommend skipping TLS verification in production environments."